### ISSUE TITLE
[$1000] Web - A non-stoping spinner is displayed when opening an image previewer carousal with a deep link

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#23899]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open a chat with a random user
2. Send a picture
3. Open the sent picture
4. Copy link form the address bar
5. Sign out
6. Paste the copied link to the address bar
7. Press enter
8. Enter the same email address you have used to send the picture
9. Click continue > enter magic code

## Expected Result:
 Picture should finish loading & should be displayed

## Actual Result:
Spinner keeps spinning without stop

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.47.3
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/93399543/cad74a63-6811-4526-8037-50c50de05111

https://github.com/Expensify/App/assets/93399543/e31f86a1-0bf1-4fd2-8569-9ee638dc07d6

**Expensify/Expensify Issue URL:**
**Issue reported by:** @Natnael-guchima
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1690629431697969

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01e0613ad8be2f9b94</li>
        <li>Upwork Job ID: 1686036587549114368</li>
        <li>Last Price Increase: 2023-07-31</li>
<li>Automatic offers: </li>
<ul>
<li>huzaifa-99 | Contributor | 25986445</li>
<li>Natnael-guchima | Reporter | 25986447</li>
</ul></li>
    </ul>
</details>

**Comment by robertjchen** [comment]
Thanks for the discussion here- having read the above, I agree that @huzaifa-99 's updated proposal would be best!

**Comment by flaviadefaria** [comment]
@mallenexpensify I'm officially no longer in the BugZero team so reassigning this as it will not close until tomorrow.

**Comment by mallenexpensify** [comment]
@allroundexperts what's the status here? I see the below as the last comment in/on [the PR](https://github.com/Expensify/App/pull/24210#issuecomment-1690705623)
<img width="420" alt="image" src="https://github.com/Expensify/App/assets/22508304/78b6e636-ef68-4fbd-8472-aa1b82f6524f">

**Comment by mallenexpensify** [comment]
Bumping to weekly since the PR is being worked on.  @allroundexperts @huzaifa-99 can you provide an update? It looks like the PR for this issue ([here](https://github.com/Expensify/App/pull/24210#issuecomment-1779056464)) was held on 
- https://github.com/Expensify/App/issues/27168 

And the PR for that hit production 3 weeks ago
- https://github.com/Expensify/App/pull/28278#issuecomment-1767338576

**Comment by mallenexpensify** [comment]
Thanks @huzaifa-99 put on hold and updated the title, pending
- https://github.com/Expensify/App/issues/27168

**Comment by mallenexpensify** [comment]
Still holding, removed `Reviewing` label bumped to weekly.

**Comment by robertjchen** [comment]
Still holding for now

**Comment by mallenexpensify** [comment]
Held still

**Comment by robertjchen** [comment]
Still holding on https://github.com/Expensify/App/issues/27168 it looks like?

**Comment by mallenexpensify** [comment]
I think... commented there cuz it was laggin a bit
https://github.com/Expensify/App/issues/27168#issuecomment-1955553153

**Comment by robertjchen** [comment]
holding on results of https://github.com/Expensify/App/issues/27168

**Comment by mallenexpensify** [comment]
Still holding.

**Comment by s77rt** [comment]
https://github.com/Expensify/App/issues/27168 is closed. Can we proceed here?

**Comment by mallenexpensify** [comment]
Off hold, added the `retest-weekly` label.  This bugs seems like an extreme edge case.  The only reason I think it'd be worth fixing is because the end result renders the app useless with a spinner.  

Are you able to reproduce @s77rt?

**Comment by s77rt** [comment]
Yes, still reproducible


https://github.com/Expensify/App/assets/16493223/dcbf1d20-77d4-4160-a4ff-aa3d8be92976

**Comment by mallenexpensify** [comment]
Thanks @s77rt , do you think the bug exists elsewhere? ie. from other types of links that are copy/pasted? With it being an edge case, if we're going to fix it, I'd love for the fix to help other instances, if they exist.

**Comment by s77rt** [comment]
@mallenexpensify Yes, I think this probably affects other links such as links to profile avatars, links to distance receipts, etc. (any image that is displayed in a modal)

**Comment by mallenexpensify** [comment]
Comment [on the PR](https://github.com/Expensify/App/pull/24210#issuecomment-2005018643)
> I think we should wait for https://github.com/Expensify/App/issues/27168 to get this resolved.
The above was closed, it was not reproducible.
Should we proceed here now?

@huzaifa-99 , if you're able to continue with this, can you test other instance like @s77rt mentioned above to see if we can fix more than just this single bug with the PR?  Thx.

**Comment by mallenexpensify** [comment]
Added to #vip-vsb
Came off hold a couple days ago, it's quite a dusty issue, since the bug was reported in July. 
Waiting on retest, we might be looking for brand new proposals

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by mallenexpensify** [comment]
Also tried to repro and wasn't able to. Closing.   Bonus, check out this bike I got last weeekend!
<img width="857" alt="image" src="https://github.com/Expensify/App/assets/22508304/b0d9ca20-8cfe-4f18-a26b-0a1b4d0890e0">
